src/test/ui/recursion/issue-95134.rs

```diff
@@ -1,28 +0,0 @@
-// build-fail
```

Comment:
This should be kept. If it ICEs, then it can be tagged as `// known-bug: ####` and you need to add `// failure-status: 101` and `// dont-check-compiler-stderr` to account for the ICE.